src/sipm_simple_calibration.jl

```diff
@@ -0,0 +1,215 @@
+"""
+    sipm_simple_calibration(pe_uncal::Array)
+
+
+Perform a simple calibration for the uncalibrated p.e. spectrum array `pe_uncal`
+using just the 1 p.e. and 2 p.e. peak positions estimated by a peakfinder.
+
+Returns 
+    * `pe_simple_cal`: array of the calibrated pe array with the simple calibration
+    * `func`: function to use for the calibration
+    * `c`: calibration factor
+    * `peakpos`: 1 p.e. and 2 p.e. peak positions in uncalibrated amplitude
+    * `h_uncal`: histogram of the uncalibrated pe array
+    * `h_calsimple`: histogram of the calibrated pe array with the simple calibration
+"""
+
```

Comment:
Can you remove the line break here? Then the docstring is properly attached to the function